api/src/main/java/au/org/aodn/nrmn/restapi/controller/StagedJobController.java

```diff
@@ -138,9 +138,9 @@
             jobRepo.save(job);
 
             List<StagedRow> rowsToSave = parsedSheet.getStagedRows();
-            Long numEmptyRows = parsedSheet.getNumEmptyRows();
+            Long skippedRows = parsedSheet.getSkippedRows();
 
-            String message = "Staged " + (rowsToSave.size()) + " row(s)." + (numEmptyRows > 0 ? " " + numEmptyRows + " empty " + "row(s) skipped." : "");
+            String message = "Staged " + (rowsToSave.size()) + " row(s)." + (skippedRows > 0 ? " " + skippedRows + " rows missing and ID but containing data were skipped." : "");
```

Comment:
"missing and ID" should be "missing an ID"?

Reply:
Fixed.